includes/widgets/heading.php

```diff
@@ -344,7 +344,7 @@ protected function content_template() {
 		var title = settings.title;
 
 		if ( '' !== settings.link.url ) {
-			title = '<a href="' + settings.link.url + '">' + title + '</a>';
+			title = '<a href="' + _.escape( settings.link.url ) + '">' + title + '</a>';
```

Comment:
```suggestion
			title = '<a href="' + _.escape( settings.link.url ) + '">' + title + '</a>';
```

check if it also escapes things like `javascript:alert(1);`

also other URLs

Reply:
URLs are not clickable in frontend preview.